tests/cstor/clone/clone_test.go

```diff
@@ -39,45 +42,50 @@ var _ = Describe("[cstor] TEST VOLUME CLONE PROVISIONING", func() {
 
 	BeforeEach(func() {
 		When("deploying cstor sparse pool", func() {
-			By("building storageclass object")
-			scObj, err = sc.NewBuilder().
-				WithName(scName).
-				WithAnnotations(annotations).
-				WithProvisioner(openebsProvisioner).Build()
-			Expect(err).ShouldNot(HaveOccurred(), "while building storageclass obj for storageclass {%s}", scName)
-
-			By("creating storageclass")
-			_, err = ops.SCClient.Create(scObj)
-			Expect(err).To(BeNil(), "while creating storageclass", scName)
-
-			By("building storagepoolclaim")
+			By("building spc object")
 			spcObj = spc.NewBuilder().
-				WithName(spcName).
+				WithGenerateName(spcName).
 				WithDiskType(string(apis.TypeSparseCPV)).
 				WithMaxPool(cstor.PoolCount).
 				WithOverProvisioning(false).
 				WithPoolType(string(apis.PoolTypeStripedCPV)).
 				Build().Object
 
-			By("creating above storagepoolclaim")
-			_, err = ops.SPCClient.Create(spcObj)
+			By("creating storagepoolclaim")
+			spcObj, err = ops.SPCClient.Create(spcObj)
 			Expect(err).To(BeNil(), "while creating spc", spcName)
 
-			By("verifying healthy cstorpool count")
-			cspCount := ops.GetHealthyCSPCount(spcName, cstor.PoolCount)
+			By("verifying healthy csp count")
+			cspCount := ops.GetHealthyCSPCount(spcObj.Name, cstor.PoolCount)
 			Expect(cspCount).To(Equal(1), "while checking cstorpool health count")
```

Comment:
Should it be: `Expect(cspCount).To(Equal(cstor.PoolCount), "while checking cstorpool health status")`